peribolos/knative-sandbox.yaml

```diff
@@ -770,6 +770,7 @@ orgs:
         members:
         - cardil
         - chaodaiG
+        - chizhg
```

Comment:
Just an FYI -- sub-teams are automatically members of the parent team, so you shouldn't need this. It doesn't hurt, though.